src/main/resources/META-INF/persistence.xml

```diff
@@ -1,13 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <persistence version="1.0" xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_1_0.xsd">
-    <persistence-unit name="VDCNet-ejbPU" transaction-type="JTA">
+    <persistence-unit name="VDCNet-ejbPU" transaction-type="RESOURCE_LOCAL">
```

Comment:
Is there a reason for changing transaction type?

Reply:
Forgot to revert to original version of persistence.xml (this is the only way for my ide to generate sql from jpa)